cpu/riscv_common/Kconfig

```diff
@@ -12,6 +12,7 @@ config CPU_ARCH_RISCV
     select HAS_NEWLIB
     select HAS_PERIPH_CORETIMER
     select HAS_PICOLIBC if '$(RIOT_CI_BUILD)' != '1'
+    #select HAS_RUST_TARGET
```

Comment:
Is this currently broken?

Reply:
Yes -- or maybe not even fully, but as with the ARMv8, I'm focusing on some to get a stable reference shape of things.